cmd/influx/backup.go

```diff
@@ -6,15 +6,15 @@ import (
 	"github.com/influxdata/influx-cli/v2/clients/backup"
 	br "github.com/influxdata/influx-cli/v2/internal/backup_restore"
 	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
-	"github.com/urfave/cli/v2"
+	"github.com/urfave/cli"
 )
 
-func newBackupCmd() *cli.Command {
```

Comment:
Mostly out of curiosity...why did these all have to change from being pointers?

Reply:
Guess it would help if I read the migration guide: https://github.com/urfave/cli/blob/master/docs/migrate-v1-to-v2.md#commands-are-now-lists-of-pointers

Although I do still wonder why that was 😃

Reply:
The types changed for some reason between V1->V2 of urfave. The migration guide is [here](https://github.com/urfave/cli/blob/master/docs/migrate-v1-to-v2.md), I effectively applied it in reverse.